bigtable/snippets/reads/read_snippets.py

```diff
@@ -0,0 +1,177 @@
+    print_row(row)
+
+
+# [END bigtable_reads_row_partial]
+# [START bigtable_reads_rows]
+def read_rows(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+
+    row_set = RowSet()
+    row_set.add_row_key(b"phone#4c410523#20190501")
+    row_set.add_row_key(b"phone#4c410523#20190502")
+
+    rows = table.read_rows(row_set=row_set)
+    for row in rows:
+        print_row(row)
+
+
+# [END bigtable_reads_rows]
+# [START bigtable_reads_row_range]
+def read_row_range(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+
+    row_set = RowSet()
+    row_set.add_row_range_from_keys(
+        start_key=b"phone#4c410523#20190501",
+        end_key=b"phone#4c410523#201906201")
+
+    rows = table.read_rows(row_set=row_set)
+    for row in rows:
+        print_row(row)
+
+
+# [END bigtable_reads_row_range]
+# [START bigtable_reads_row_ranges]
+def read_row_ranges(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+
+    row_set = RowSet()
+    row_set.add_row_range_from_keys(
+        start_key=b"phone#4c410523#20190501",
+        end_key=b"phone#4c410523#201906201")
+    row_set.add_row_range_from_keys(
+        start_key=b"phone#5c10102#20190501",
+        end_key=b"phone#5c10102#201906201")
+
+    rows = table.read_rows(row_set=row_set)
+    for row in rows:
+        print_row(row)
+
+
+# [END bigtable_reads_row_ranges]
+# [START bigtable_reads_prefix]
+def read_prefix(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+    prefix = "phone#"
+    end_key = prefix[:-1] + chr(ord(prefix[-1]) + 1)
+
+    row_set = RowSet()
+    row_set.add_row_range_from_keys(prefix.encode("utf-8"),
+                                    end_key.encode("utf-8"))
+
+    rows = table.read_rows(row_set=row_set)
+    for row in rows:
+        print_row(row)
+
+
+# [END bigtable_reads_prefix]
+# [START bigtable_reads_filter]
+def read_filter(project_id, instance_id, table_id):
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+
+    rows = table.read_rows(filter_=ValueRegexFilter(b"PQ2A.*$"))
+    for row in rows:
+        print_row(row)
+
+
+# [END bigtable_reads_filter]
+
+
+# [START bigtable_reads_row]
+# [START bigtable_reads_row_partial]
+# [START bigtable_reads_rows]
+# [START bigtable_reads_row_range]
+# [START bigtable_reads_row_ranges]
+# [START bigtable_reads_prefix]
+# [START bigtable_reads_filter]
+def print_row(row):
```

Comment:
Would this function be useful in the library? I see it used across two snippets.

Reply:
Only for running snippets like these. So perhaps, but probably not necessary.